### ISSUE TITLE
About recurrent neural network using disout

### ISSUE THREAD

**Issue by qq260612718** [huawei-noah/Disout#11]
Can anyone tell me how to set the hyperparameters of disout in the recurrent neural network?

**Comment by yehuitang** [comment]
Thanks for your question. We mainly conduct experiments on CNNs.  For RNN, the hyper-parameters 'dist_prob' and 'alpha' need to be adjusted by watching the accuracies on training set and validation set.
